numba/tests/test_record_dtype.py

```diff
@@ -1077,5 +1103,71 @@ def test_error_w_invalid_field(self):
                       "('first', 'second')", str(raises.exception))
 
 
+class TestRecordArraySetItem(unittest.TestCase):
+    """
+    Test getitem when index is Literal[str]
+    """
+    def test_literal_variable(self):
+        arr = np.array([1, 2], dtype=recordtype2)
+        pyfunc = set_field1
+        jitfunc = njit(pyfunc)
+        self.assertEqual(pyfunc(arr[0]), jitfunc(arr[0]))
+
+    def test_literal_unroll(self):
+        arr = np.array([1, 2], dtype=recordtype2)
+        pyfunc = set_field2
+        jitfunc = njit(pyfunc)
+        self.assertEqual(pyfunc(arr[0]), jitfunc(arr[0]))
+
+    def test_literal_variable_global_tuple(self):
+        """
+        This tests the getitem of record array when the indexes come from a
```

Comment:
```suggestion
        This tests the setitem of record array when the indexes come from a
```

Reply:
In fact, all the `getitems` in the proposed test docstrings need to be `setitems` ?